hack/retag-multi-arch-images.sh

```diff
@@ -19,15 +19,17 @@ if [[ -z ${NEW_TAG} ]]; then
   exit 1
 fi
 
+. ./hack/cri-bin.sh && export CRI_BIN=${CRI_BIN}
+
 if [[ "${MULTIARCH}" == "true" ]]; then
   for arch in ${ARCHITECTURES}; do
     NEW_IMAGE="${NEW_IMAGE_REPO}:${NEW_TAG}-${arch}"
-    . "hack/cri-bin.sh" && ${CRI_BIN} tag "${IMAGE_REPO}:${CURRENT_TAG}-${arch}" "${NEW_IMAGE}"
-    . "hack/cri-bin.sh" && ${CRI_BIN} push "${NEW_IMAGE}"
+    ${CRI_BIN} tag "${IMAGE_REPO}:${CURRENT_TAG}-${arch}" "${NEW_IMAGE}"
+    ./hack/retry.sh 3 10 "${CRI_BIN} push ${NEW_IMAGE}"
```

Comment:
what does `3 10` means?

Reply:
It's in the ./hack/retry.sh script. The first parameter is the number of retries. The 2nd is the number of seconds to wait for the next attempt, in case of failure. The 3rd parameter is the command to run.

https://github.com/kubevirt/hyperconverged-cluster-operator/blob/366678ebec7bf6dbc207db72293f70a37fc744bd/hack/retry.sh#L8-L11